src/ripple/protocol/impl/TxFormats.cpp

```diff
@@ -84,6 +84,13 @@ TxFormats::TxFormats ()
     add ("TicketCancel", ttTICKET_CANCEL)
         << SOElement (sfTicketID,            SOE_REQUIRED)
         ;
+
+    // The SignerEntries are optional because a SignerList is deleted by
+    // setting the SignerQuorum to zero and omitting SignerEntries.
+    add ("SignerListSet", ttSIGNER_LIST_SET)
```

Comment:
I guess this bird has already flown, (tickets/offers etc) but using SignerListSet and SetSignerList in the same codebase seems unfortunate. Makes it a touch harder to grep the code/logs.